docker-for-mac/troubleshoot.md

```diff
@@ -6,77 +6,106 @@ redirect_from:
 title: Logs and troubleshooting
 ---
 
-Here is information about how to diagnose and troubleshoot problems, send logs
-and communicate with the Docker Desktop for Mac team, use our forums and Knowledge Hub,
-browse and log issues on GitHub, and find workarounds for known problems.
+This page contains information on how to diagnose and troubleshoot Docker Desktop issues, send logs and communicate with the Docker Desktop team, use our forums and Success Center, browse and log issues on GitHub, and find workarounds for known problems.
 
-## Docker Knowledge Hub
+## Docker Success Center
 
-**Looking for help with Docker Desktop for Mac?** Check out the [Docker Knowledge
-Hub](http://success.docker.com/) for knowledge base articles, FAQs, and
+**Looking for help with Docker Desktop on Mac?** Check out the [Docker Success Center](http://success.docker.com/) for knowledge base articles, FAQs, and
 technical support for various subscription levels.
 
+## Troubleshoot
+
+Choose ![whale menu](images/whale-x.png){: .inline} > **Troubleshoot**
+from the menu bar to see the troubleshoot options.
+
+![Uninstall or reset Docker](images/menu/troubleshoot.png){: .with-border}
+
+The Troubleshoot page contains the following options:
+
+* **Restart Docker Desktop** - Select to restart Docker Desktop.
+
+* **Run Diagnostics** - Select this option to diagnose any issues on Docker Desktop. For detailed information about diagnostics, see [Diagnose problems, send feedback, and create GitHub issues](#diagnose-problems-send-feedback-and-create-github-issues)
+
+* **Reset disk image** - This option resets all Docker data _without_ a
+reset to factory defaults. Selecting this option results in the loss of existing settings.
+
+* **Reset to factory defaults** - Choose this option to reset all options on
+Docker Desktop to its initial state, the same as when it was first installed.
```

Comment:
"to their initial state,..." reads better, but not sure if that changes the meaning. Sentence could be "Choose this option to reset all options on Docker Desktop to their initial state, the same as when Docker Desktop was first installed."

Reply:
Sounds good. Added Docker Desktop for clarity.